sk0-front/src/app/app.component.ts

```diff
@@ -9,5 +10,9 @@ import { IonicModule } from '@ionic/angular';
   imports: [IonicModule],
 })
 export class AppComponent {
-  constructor() {}
+  constructor(private authService: AuthService) {}
+
+  get userName() {
+    return this.authService.userName;
+  }
```

Comment:
I think it is different in the dev bev branch. There should be a BehavorSubject in the dev branch.

```suggestion
  userName$ = this.authService.userName
  
  constructor(private authService: AuthService) {}
```